test/tstLinearBVH.cpp

```diff
@@ -740,9 +806,8 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(structured_grid, DeviceType, ARBORX_DEVICE_TYPES)
 
   ArborX::BVH<DeviceType> bvh(bounding_boxes);
 
-  // (i) use same objects for the queries than the objects we constructed the
-  // BVH
-  // i-2  i-1  i  i+1
+  // (i) use same objects for the queries than the objects we constructed
+  // the BVH i-2  i-1  i  i+1
```

Comment:
Undo this change

Reply:
I did not do it manually, it was `clang-format`.

Reply:
Well you tell clang-format I don't like it and revert :p

Reply:
OK